applications/ConvectionDiffusionApplication/tests/test_convection_diffusion_explicit_element.py

```diff
@@ -0,0 +1,96 @@
+from KratosMultiphysics import *
+import KratosMultiphysics.ConvectionDiffusionApplication
+from KratosMultiphysics.ConvectionDiffusionApplication.convection_diffusion_analysis import ConvectionDiffusionAnalysis
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+from math import *
+
+class TestConvectionDiffusionExplicitElement(KratosUnittest.TestCase):
```

Comment:
I came up with the idea of calling this `TestConvectionDiffusionBar` so we can reuse this same test also for the non-explicit elements. What do you think? 

It should be as easy as adding more `testMyElement` functions together with the corresponding `ProjectParameters.json` files. We can add the implicit element tests in a future PR.

Reply:
Ok, done.